services/app-api/src/emailer/templates.test.ts

```diff
@@ -245,6 +242,25 @@ describe('Email templates', () => {
                 })
             )
         })
+        it('includes ratesReviewSharedEmails on contract and rate submission', () => {
```

Comment:
lets also add a tests for the opposite case (does not include rates emails when its contract only) to have a good sanity check here